velox/common/base/BloomFilter.h

```diff
@@ -129,6 +135,58 @@ class BloomFilter {
     return mask == (bloom[index] & mask);
   }
 
+  inline static void testForBatch(
+      const uint64_t* FOLLY_NONNULL bloom,
+      int32_t bloomSize,
+      const uint64_t* hashCode,
+      bool* result,
```

Comment:
`result` here should be a `uint64_t*` bit mask, then you can directly convert the comparison result to bit mask using `toBitMask` in `SimdUtil.h` and write directly into the `FlatVector<bool>` without row wise conversion (which is quite expensive and can be one of the bottleneck after you simdizing it).